frontend/src/components/buildings/Buildings.js

```diff
@@ -1,11 +1,73 @@
 import React from 'react';
-import AddBuilding from './AddBuilding';
 import './Buildings.css';
+import { connect } from 'react-redux';
+import farm from '../../assets/buildings/farm.svg';
+import mine from '../../assets/buildings/mine.svg';
+import townhall from '../../assets/buildings/townhall.svg';
+import academy from '../../assets/buildings/barracks.svg';
+import { setErrorAction } from '../../actions/ErrorActions';
+import { getBuildingsAction} from '../../actions/BuildingsActions';
 
-export default function Buildings() {
+
+function Buildings(props){
+      
+        const buildingIcons = {
+          farm: farm,
+          mine: mine,
+          townhall: townhall,
+          academy: academy,
+        };
+      
+
+function buildingLevel({kingdom}){
+
+}
+        
   return (
-    <div className="buildings">
-      <AddBuilding />
-    </div>
+    <section className="buildings">
+        
+      <div className="icons">
+        <img src={buildingIcons.townhall} alt="townhall" className="image"></img>
+        <p className="buildingName">Townhall</p> 
+        <p>{buildingLevel.townhall}</p>
+      </div>
```

Comment:
@zvorsi this should be a building, so looping through the building this should be displayed multiple times:
```suggestion
  { buildings.map(building => (
      <div className="icons" onClick={() => goToBuilding(building.id)}>
        <img src={buildingIcons[building.type]} alt={building.type} className="image"></img>
        <p className="buildingName">{building.name}</p> 
        <p>{building.level}</p>
      </div>
      ))}
```